ckanext/canada/schemas/presets.yaml

```
@@ -3336,6 +3336,9 @@ presets:
       label: GeoSoftGrids
     - value: GRIB1
     - value: GRIB2
+    - value: Gzip
```

I think this is the correct extension
```suggestion
    - value: GZ
```